examples/remix-basic/app/match-stream.ts

```diff
@@ -0,0 +1,42 @@
+import { ShapeStream, ChangeMessage } from "@electric-sql/next"
+
+export async function matchStream({
+  stream,
+  operations,
+  matchFn,
+  timeout = 10000,
+}: {
+  stream: ShapeStream
+  operations: Array<`insert` | `update` | `delete`>
```

Comment:
hmm yes we should be consistent but... `operation` seems like the right word vs. `action` (to stay consistent with Postgres terminology). So updating everything to operation makes sense. Actions seems to only be used as a word for work done because a rule is triggered.

Reply:
https://github.com/electric-sql/electric-next/issues/252